atomic_qt_design/qml/Exchange/History/History.qml

```diff
@@ -63,18 +41,12 @@ Item {
 
         SwapList {
             title: API.get().empty_string + (qsTr("Recent Swaps"))
-            items: getRecentSwaps()
+            items: API.get().orders_mdl
```

Comment:
why this is not using the proxy ?

Reply:
It's not the `ListView`, it's the wrapper. This takes the `orders_mdl` so it can use `.length` inside, and feeding `proxy` into the real `ListView`.